src/flick/user/serializers.py

```diff
@@ -61,3 +63,39 @@ class Meta:
             "collab_lsts",
         )
         read_only_fields = fields
+
+
+class FriendProfileSerializer(serializers.ModelSerializer):
+    profile_id = serializers.CharField(source="id")
+    user_id = serializers.CharField(source="user.id")
+    first_name = serializers.CharField(source="user.first_name")
+    last_name = serializers.CharField(source="user.last_name")
+    username = serializers.CharField(source="user.username")
+    profile_pic = AssetBundleDetailSerializer(source="profile_asset_bundle")
+    public_owner_lst = serializers.SerializerMethodField("public_owner_lsts")
+    public_collab_lst = serializers.SerializerMethodField("public_collab_lsts")
```

Comment:
instead of these, let's keep naming consistent with `owner_lsts` and `collab_lsts` like we have at `/api/auth/me`